CefSharp.Core/Internals/ClientAdapter.h

```diff
@@ -62,6 +62,8 @@ namespace CefSharp
             void CloseDevTools();
             void CloseAllPopups(bool forceClose);
 
+            virtual bool IsOffscreen() { return false; }
```

Comment:
This name is  slightly misleading. Shouldn't it rather be something like `IsPopupInSeparateWebBrowserControl` or something? The parameter documentation for `newBrowser` just says its in a different control, `newBrowser` doesn't need to necessarily be an Offscreen control.